packages/builders/scripts/vats/init-orchestration.js

```diff
@@ -1,9 +1,19 @@
 import { makeHelpers } from '@agoric/deploy-script-support';
 
-/** @type {import('@agoric/deploy-script-support/src/externalTypes.js').ProposalBuilder} */
+// If we like this, move it into deploy-script-support.
+// Consider also `.permit.json` to distinguish the module shape qualifier ('permit') from the identifier. (Similarly `.plan.json`.)
+const prefix = (evalName = '') => {
+  const proposalPath = import.meta.url;
+  const left = proposalPath.lastIndexOf('/') + 1;
+  const right = proposalPath.lastIndexOf('.');
+  const filename = proposalPath.substring(left, right);
+  return `${filename}-${evalName}`;
+};
```

Comment:
These names seems misleading to me, since the behavior is actually to _suffix_ a base component.
```suggestion
// If we like this, move it into deploy-script-support.
// Consider also `.permit.json` to distinguish the module shape qualifier ('permit') from the identifier. (Similarly `.plan.json`.)
const getBaseName = (keepExtension = false) => {
  const proposalPath = import.meta.url;
  const baseName = proposalPath.split('/').at(-1);
  return keepExtension ? baseName : baseName.replace(/[.][^.]*$/, '');
};
```
with
```js
  await writeCoreEval(`${getBaseName()}-gov`, defaultProposalBuilder);
```

Reply:
The idea was "this generates the prefix for writeCoreEval". If we were to move the function out, it would be just,
```
const getBaseName = (proposalPath, keepExtension = false) => {
  const baseName = proposalPath.split('/').at(-1);
  return keepExtension ? baseName : baseName.replace(/[.][^.]*$/, '');
};
```
because we can import `import.meta.url`. At that point it's basically Node's `path.basename`. I'll drop this commit too and push a separate PR to hash out the design.

Reply:
https://github.com/Agoric/agoric-sdk/pull/9472